.cbmc-batch/jobs/aws_cryptosdk_md_abort/Makefile

```diff
@@ -35,6 +36,11 @@ DEPENDENCIES += $(SRCDIR)/helper-src/openssl/bn_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/ec_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/evp_override.goto
 
+REMOVE_FUNCTION_BODIES += --remove-function-body EVP_MD_CTX_free
+REMOVE_FUNCTION_BODIES += --remove-function-body evp_md_ctx_is_valid
```

Comment:
why

Reply:
Does it cause the proof to hang to validate the evp_md_ctx due to lots of processing? If so this makes sense, just add a comment explaining that.

Reply:
Same as in https://github.com/aws/aws-encryption-sdk-c/pull/582#discussion_r478660427. This has been updated!